src/index.tsx

```diff
@@ -71,7 +73,9 @@ type TextAreaValidationAttrs = Pick<
   "required" | "minLength" | "maxLength"
 >;
 
-type SelectValidationAttrs = Pick<BuiltInValidationAttrs, "required">;
+type SelectValidationAttrs = {
+  multiple?: boolean;
+} & Pick<BuiltInValidationAttrs, "required">;
```

Comment:
Added support for `multiple` attribute on `select`

---

```html
<select name="a">
<select name="a">
<select name="a">
```
```ts
inputs: {
  a: {
    element: "select",
    multiple: true
  }
}
```

---

```html
<select name="a" multiple>
```
```ts
inputs: {
  a: {
    element: "select",
    validationAttrs: {
      multiple: true
    }
  }
}
```

---

```html
<select name="a">
<select name="a" multiple>
```
```ts
inputs: {
  a: {
    element: "select",
    multiple: true
    validationAttrs: {
      multiple: true
    }
  }
}
```